Karen_dinner_party.rb

```diff
@@ -0,0 +1,65 @@
+
+#use for a pre-populated party with guest, food, and bevs. Otherwise use user input option below.
+#party_elements = {beverages: ["coke", "diet coke", "sprite", "tequila", "tequiza", "zima", "bud light lime", "orange juice", "water", "coffee", "tea", "jim beam", "cerveza"], food: ["porkchop", "veggie nuggets", "shrimp", "spaghetti"], people:
+#["Karen", "Wolfie", "Erin", "Tim", "Thom", "Stephen", "Ken", "James"], seat_num:[]}
+
+#user input arrays
+party_elements = {beverages: [], food: [], people:
```

Comment:
If a `Hash` doesn't fit on a single line, do a line for each key/value with indentation:
```rb
party_elements = {
  beverages: [],
  food: [],
  people: [],
  seat_num: []
}
```